packages/forms/demo/stories/FileStory.tsx

```diff
@@ -10,10 +10,12 @@ import { Story } from '@storybook/react';
 import { Progress } from '@zendeskgarden/react-loaders';
 import { File, IFileProps } from '@zendeskgarden/react-forms';
 
-interface IArgs extends IFileProps {
+interface IArgs extends Omit<IFileProps, 'onClick'> {
   hasClose?: boolean;
   hasDelete?: boolean;
   value?: number;
+  onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
```

Comment:
Should `onClick` have been added to the story interface? I'm not seeing the need for it below.

Reply:
It's used if hasClose/hasDelete are `true`. 

https://github.com/zendeskgarden/react-components/pull/1509/files/f4dd3066eb8c485a22058dfff2be41c0f0561ee7#diff-e827e562c50bd447dc80badf1cf1c86d052c30671b9e1dbb064761bef2040fe4R33

Reply:
How do you mean?

Reply:
Oh, I get it now – it's no longer a `<div>` click, but rather a `<button>` click. In that case, perhaps the same as below?

```
  onClick?: ButtonHTMLAttributes<HTMLButtonElement>['onClick'];
```

Reply:
Fixed: 2240a0c47df42d203687952837d6ef7d4af79886